cmd/ocm/main.go

```diff
@@ -96,7 +97,20 @@ func main() {
 		fmt.Fprintf(os.Stderr, "Can't parse empty command line to satisfy 'glog': %v\n", err)
 		os.Exit(1)
 	}
+	args := os.Args
+	pluginHandler := plugin.NewDefaultPluginHandler([]string{"ocm"})
+	if len(args) > 1 {
+		cmdPathPieces := args[1:]
 
+		// only look for suitable extension executables if
+		// the specified command does not already exist
+		if _, _, err := root.Find(cmdPathPieces); err != nil {
+			if err := plugin.HandlePluginCommand(pluginHandler, cmdPathPieces); err != nil {
+				fmt.Fprintf(os.Stderr, "%v\n", err)
+				os.Exit(1)
+			}
```

Comment:
If there was no error, this proceeds to execute built-in command.
in most scenarios, if a plugin was found HandlePluginCommand() will not return at all, but i think on windows it currently _may_ return if plugin run succesfully and returned 0.  

Not saying you should handle this here, just pointing out this scenario is now possible.  See comments on plugin.go on pinning down the contract of HandlePluginCommand()...

Reply:
resolved by last changes :heavy_check_mark: